Exercise_1/index.html

```diff
@@ -0,0 +1,128 @@
+<!doctype html>
+<html lang="pl">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+<table width="100%" border="1" cellpadding="5px">
+    <thead>
+    <tr>
+        <th colspan="3" height="100px">
+            <h1>Guide to the City</h1>
+        </th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td valign="top" width="150px">
+            <h2 align="center">
```

Comment:
You could wrap it in `<header>` tag. Your solution is ok but semantically it would be better.